docs/en/start-v2/docker/docker.md

```diff
@@ -139,3 +138,227 @@ docker run --rm -it apache/seatunnel bash -c '<YOUR_FLINK_HOME>/bin/start-cluste
 docker run --rm -it apache/seatunnel bash -c '<YOUR_FLINK_HOME>/bin/start-cluster.sh && ./bin/start-seatunnel-flink-15-connector-v2.sh -c config/v2.streaming.conf.template'
 ```
 
+
+
+# Set Up With Docker In Cluster Mode
+
+there has 2 ways to create cluster within docker.
+
+## 1. Use Docker-compose
+
+> docker cluster mode is only support zeta engine.
+
+The `docker-compose.yaml` file is :
+```yaml
+version: '3.8'
+
+services:
+  master:
+    image: apache/seatunnel
+    container_name: seatunnel_master
+    environment:
+      - ST_DOCKER_MEMBER_LIST=172.16.0.2,172.16.0.3,172.16.0.4    
```

Comment:
where can user get `ST_DOCKER_MEMBER_LIST` value?

Reply:
in docker-compose, we can define each container's ip. so we can know the ip list and set it to here